ddtrace/internal/_encoding.pyx

```diff
@@ -212,65 +199,65 @@ cdef class Packer(object):
         ret = msgpack_pack_map(&self.pk, L)
 
         if ret == 0:
-            ret = pack_bytes(&self.pk, <char *> b"trace_id", 8)
+            ret = msgpack_pack_write(&self.pk, <char *> b"\xa8trace_id", 9)
```

Comment:
this is really a micro-optimization, on it's own it didn't yield a massive gain, but I figured it is easy/doesn't hurt to keep.

Reply:
IMHO if we don't really gain too much out of this, we might want to leave implementation details such as `\xa8` to the `msgpack` API

Reply:
I was 50/50 on it, but since the values are effectively static, and will never change in the lifetime of msgpack, saving on a few operations is always a benefit for sure.

no strong opinion, we will need to re-think this with v0.5 anyways since the encoding doesn't even encode a span as a map, but as an array, so we won't even have these